### PR TITLE
009: improvements

### PR DESCRIPTION
**Pull request by liranh85**

### CODE REVIEW

File: examples/009/log-color.js [hunk 1]
```diff
@@ -1,8 +1,7 @@
...
 
 require('colors');
-const { log } = require('console');
 
-log('This is a success message'.green);
-log('This is a warning message'.yellow);
-log('This is a error message'.red);
+console.log('This is a success message'.green);
+console.log('This is a warning message'.yellow);
+console.log('This is a error message'.red);
```
Comment (smks) [inline examples/009/log-color.js]: What's the reason for this change?

Comment (liranh85) [reply examples/009/log-color.js]: Two reasons:
1) There is no need to require the console, it's a global variable
2) extracting out log from the global console and using it on its own might confuse readers, as it looks exactly the same as your custom log function.
